keps/sig-network/1672-tracking-terminating-endpoints/kep.yaml

```diff
@@ -20,16 +20,18 @@ see-also:
 replaces: []
 
 # The target maturity stage in the current dev cycle for this KEP.
-stage: alpha
+stage: stable
```

Comment:
Please fill in the PRR questionaire for this KEP.

Reply:
hmm there's no diff for PRR cause all the questions were answered in the last release, let me know if there's a specific question I missed.

Reply:
There should be a diff - there are couple questions that weren't answered back then, including:

> Are there any tests for feature enablement/disablement?

Have the tests been added? If so, please link them. If not - we shouldn't even go to beta...

> What specific metrics should inform a rollback?

We were discussing adding a metric - has that happened?

> Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?

Has that happened? Findings?

> What are the SLIs (Service Level Indicators) an operator can use to determine the health of the service?

Has the metric mentioned there been added?

Reply:
> Have the tests been added? If so, please link them. If not - we shouldn't even go to beta...

Oh yes, of course we added tests for feature enablement, do we want the PRR to link to every test case we added? The answer to the question says:
```
Yes, there will be strategy API unit tests validating if the new API field is allowed based on the feature gate.
``` 

Is that enough or do you want more details on the specific test cases (there's a lot)?

Reply:
Let me add more details on metrics, I think there was some back and forth on the viability of per endpoint metrics due to cardinality. But maybe total endpoints by condition is acceptable.

Reply:
> Is that enough or do you want more details on the specific test cases (there's a lot)?

Please link them - no need to describe.

Reply:
Added all the PRs that adds unit, integration or e2e tests for this feature under `Are there any tests for feature enablement/disablement?`